cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/PythonActionTransformer.java

```diff
@@ -17,15 +17,13 @@
 import com.google.common.collect.Sets;
 import io.cloudslang.lang.compiler.SlangTextualKeys;
 import java.io.Serializable;
-import java.util.Collections;
-import java.util.Set;
-import org.springframework.stereotype.Component;
+import java.util.*;
 
-import java.util.Map;
-import java.util.List;
+import org.springframework.stereotype.Component;
 
 @Component
 public class PythonActionTransformer extends AbstractTransformer implements Transformer<Map<String, Serializable>, Map<String, Serializable>> {
+    private static DependencyFormatValidator dependencyFormatValidator = new DependencyFormatValidator();
```

Comment:
I think we could define this class as a spring bean (@Component), the autowire it (if you don't go with the `PreCompileValidator` approach)